code/script.js

```diff
@@ -1,47 +1,146 @@
 
-// Global variables, if you need any, declared here
+let questionNumber = 1
 
-// Functions declared here
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
+
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
-// This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user.jpg" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot.jpg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => showMoods(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => showType(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => showGIF(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(thankYou, 1000)
+  }
+}
+
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
+  botReply(`Hello there, what's your name?`)
+}
+
+const showMoods = (msg) => {
+  questionNumber++
+  botReply(
+    `Nice to meet you ${msg}. How do you feel?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="goodBtn">Good</button>
+    <button id="mehBtn">Meh</button>
+    <button id="badBtn">Bad</button>
+  `
+
+  document
+    .getElementById('goodBtn')
+    .addEventListener('click', () => nextQuestion('good'))
+  document
+    .getElementById('mehBtn')
+    .addEventListener('click', () => nextQuestion('meh'))
+  document
+    .getElementById('badBtn')
+    .addEventListener('click', () => nextQuestion('bad'))
+}
+
+const showType = (type) => {
+  questionNumber++
+
+  botReply(
+    `Oh so you're in a ${type} mood? Can I help you with something?`
+  )
+
+ inputWrapper.innerHTML = `
+    <button id="Help me">Help me</button>
+    <button id="Maybe">Maybe</button>
+  `
+  document
+    .getElementById('Help me')
+    .addEventListener('click', () => nextQuestion('Help me'))
+  document
+    .getElementById('Maybe')
+    .addEventListener('click', () => nextQuestion('Maybe'))
+}
+
+const showGIF = (size) => {
```

Comment:
The name of this param doesn't make any sense. Maybe just `answer` would be ok here?